pkg/util/util.go

```diff
@@ -229,3 +233,128 @@ func SplitQualifiedName(str string) (string, string) {
 func JoinQualifiedName(namespace, name string) string {
 	return path.Join(namespace, name)
 }
+
+type Route struct {
+	Interface   string
+	Destination net.IP
+	Gateway     net.IP
+	// TODO: add more fields here if needed
+}
+
+func getRoutes(file io.Reader) ([]Route, error) {
+	routes := []Route{}
+	scanner := bufio.NewReader(file)
+	for {
+		line, err := scanner.ReadString('\n')
+		if strings.HasPrefix(line, "Destination") || strings.HasPrefix(line, "Kernel") {
+			continue
+		}
+		if err == io.EOF {
+			break
+		}
+		fields := strings.Fields(line)
+		if len(fields) < 8 {
+			return nil, fmt.Errorf("Incorrect number of fields (expected at least 8, got %d): %s", len(fields), line)
+		}
+		routes = append(routes, Route{})
+		route := &routes[len(routes)-1]
+		route.Interface = fields[7]
+		ip := net.ParseIP(fields[0])
+		if ip == nil {
+			return nil, fmt.Errorf("Unable to Parse IP: %s", fields[0])
+		}
+		route.Destination = ip
+		ip = net.ParseIP(fields[1])
+		if ip == nil {
+			return nil, fmt.Errorf("Unable to Parse IP: %s", fields[1])
+		}
+		route.Gateway = ip
+	}
+	return routes, nil
+}
+
+func getRouteTables() (string, error) {
+	args := []string{"-n"}
+	out, err := exec.Command("route", args...).Output()
```

Comment:
I'm a little afraid of the potential size of this, which is why I was using a proc file.  Am I just being paranoid?  I'm willing to be convinced...